### DISCUSSION TITLE
My Fork

### DISCUSSION THREAD

**Discussion by im-parsa** [IntelligentQuantum/IntelligentQuantum.GitHub.IO#24] [Polls]
Kako man neshastam reac to nextesh kardam nice bede bere

**Comment by im-parsa** [comment]
nice bemola

**Comment by IntelligentQuantum** [comment]
Hala man fekramo konam bebinam server niyaz darim ya na

**Reply by im-parsa** [reply]
are fekr kon fekr kardan kheili moheme

**Reply by IntelligentQuantum** [reply]
> are fekr kon fekr kardan kheili moheme

Ha kako nayam bala saret

**Reply by paraffin-tutorials-org** [reply]
goonah nadare ke gorbah

**Comment by paraffin-tutorials-org** [comment]
are manam ba php mizanam

**Reply by im-parsa** [reply]
vala bekhoda zendegi sakhti shode
